uid-pages/layout-bonita/test/specs/layoutBonita.feature

```diff
@@ -0,0 +1,6 @@
+Feature: The bonita layout v1
```

Comment:
maybe "Bonita layout"
Do we need really the version here ? Because I think we might forget to update this file when creating a new version